src/View/Helper/PaginatorHelper.php

```diff
@@ -1121,4 +1121,31 @@ public function implementedEvents()
     {
         return [];
     }
+
+    /**
+     * Dropdown array for Select limit
+     *
+     * @param array $limits This is array of options.
+     * @param array $default Default limit for option selecting. Default is Paginator limit.
+     *
+     * @return string html output.
+     */
+    public function showLimit($limits = [], $default = 0)
+    {
+        $out = $this->Form->create('', ['type' => 'get']);
+
+        if (empty($limit)) {
+            $limits = [10 => $this->Number->format(10), 25 => $this->Number->format(25), 50 => $this->Number->format(50), 100 => $this->Number->format(100)];
```

Comment:
Shouldn't the default value (and selected) be the default 20 coming from the component?